packages/sanity/src/core/changeIndicators/constants.ts

```diff
@@ -1,6 +1,3 @@
-// Toggle tracking of change indicator DOM positions and connectors overlay rendering
-export const ENABLED = true
```

Comment:
This might be my lack of context of this overall section showing, but is it safe then to remove it? I see that you've removed the use of it everywhere else, but I'm wondering how far impacting this is

Especially since before we were using this at least to have an alternative to when there is no presence set up, right?

Reply:
The original intent seems to have been to provide an easy way to ship a new version of `sanity` that rolls back the new UI, `git blame` says it was introduced 4 years ago and it's never been set to `false`. It feels like now is the time to delete the unused code :) 

If we were to somehow need it it's still possible to get the same result by commenting out `<ChangeIndicatorsTracker>`: https://github.com/sanity-io/sanity/blob/edc930114e34467dff7969a12039eda134843406/packages/sanity/src/core/changeIndicators/overlay/ChangeConnectorRoot.tsx#L40